src/Core/Components/Forms/FluentValidationSummary.razor

```diff
@@ -0,0 +1,21 @@
+﻿@namespace Microsoft.Fast.Components.FluentUI
+@using Microsoft.AspNetCore.Components.Forms
+@inherits ValidationSummary
+
+@{ var validationMessages = Model is null ? FluentEditContext.GetValidationMessages() : FluentEditContext.GetValidationMessages(new FieldIdentifier(Model, string.Empty));
```

Comment:
This line of code should be in razor.cs file

Reply:
Done